docs/man_pages/publishing/appmanager-upload-ios.md

```diff
@@ -3,7 +3,7 @@ appmanager upload ios
 
```

Comment:
I must update appmanager-upload-android/wp8  help in the sammer right? Also change the description for 'app manager group' command in the Related Commands with the message from your prevoius comment?